Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -176,9 +176,14 @@ void OnEditingChanged(object sender, EventArgs eventArgs)
 		void OnEditingEnded(object sender, EventArgs e)
 		{
 			// Typing aid changes don't always raise EditingChanged event
-			if (Control.Text != Element.Text)
+
+			// Normalizing nulls to string.Empty allows us to ensure that a change from null to "" doesn't result in a change event.
+			// While technically this is a difference it serves no functional good.
+			var controlText = Control.Text ?? string.Empty;
```

Comment:
For the other hand, there may be some projects where it is necessary to notify when value changes from null to string empty

Reply:
Thats why I only did this one way from renderer to source. It will notify the renderer when you change it yourself.